lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
@davidtheclark doesn't work :( I've tried to put `jest.mock("os", () => ({ EOL: "sdfsdfsfd" }))` in:

* `/lib/rules/comment-empty-line-before/__tests__/index.js`
* `jest-setup.js` in `describe` function.
* `jest-setup.js` in `spec` (`it` alias) function.